__tests__/App-test.js

```diff
@@ -1,14 +1,215 @@
+import {ImageExamplePage} from '../src/examples/ImageExamplePage';
+import {PressableExamplePage} from '../src/examples/PressableExamplePage';
+import {FlatListExamplePage} from '../src/examples/FlatListExamplePage';
+import {ScrollViewExamplePage} from '../src/examples/ScrollViewExample';
+import {PopupExamplePage} from '../src/examples/PopupExamplePage';
+import {FlyoutExamplePage} from '../src/examples/FlyoutExamplePage';
+import {CheckBoxExamplePage} from '../src/examples/CheckBoxExamplePage';
+import {ConfigExamplePage} from '../src/examples/ConfigExamplePage';
+import {DatePickerExamplePage} from '../src/examples/DatePickerExamplePage';
+import {TimePickerExamplePage} from '../src/examples/TimePickerExamplePage';
+//import {SketchExamplePage} from '../src/examples/SketchExamplePage';
+import {SliderExamplePage} from '../src/examples/SliderExamplePage';
+import {PermissionsExamplePage} from '../src/examples/PermissionsExamplePage';
+import {PickerExamplePage} from '../src/examples/PickerExamplePage';
+import {PrintExamplePage} from '../src/examples/PrintExamplePage';
+import {DeviceInfoExamplePage} from '../src/examples/DeviceInfoExamplePage';
+import {TTSExamplePage} from '../src/examples/TTSExamplePage';
+import {WebViewExamplePage} from '../src/examples/WebViewExamplePage';
+import {SensitiveInfoExamplePage} from '../src/examples/SensitiveInfoExamplePage';
+import {ProgressViewExamplePage} from '../src/examples/ProgressViewExamplePage';
+import {XamlExamplePage} from '../src/examples/XamlExamplePage';
+//import {TrackPlayerExamplePage} from '../src/examples/TrackPlayerExamplePage';
+import {WindowsHelloExamplePage} from '../src/examples/WindowsHelloExamplePage';
+import {ExpanderExamplePage} from '../src/examples/ExpanderExamplePage';
+import {View} from 'react-native';
+
+function Control(props) {
+  return <View />;
+}
+
+test('Control', () => {
+  const tree = create(<Control />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+test('Button Example Page', () => {
+  const tree = create(<ButtonExamplePage />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+test('CheckBox Example Page', () => {
+  const tree = create(<CheckBoxExamplePage />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+test('Config Example Page', () => {
+  const tree = create(<ConfigExamplePage />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+test('DatePicker Example Page', async () => {
+  let tree;
+  await act(() => {
+    tree = create(<DatePickerExamplePage />).toJSON();
+  });
+  expect(tree).toMatchSnapshot();
+});
+
+test('DeviceInfo Example Page', async () => {
+  let tree;
+  await act(() => {
+    tree = create(<DeviceInfoExamplePage />);
+  });
+  expect(tree.toJSON()).toMatchSnapshot();
+});
+
+test('Expander Example Page', async () => {
+  let tree;
+  await act(() => {
+    tree = create(<ExpanderExamplePage />);
+  });
+  expect(tree.toJSON()).toMatchSnapshot();
+});
+
+/*test('Permissions Example Page', async () => {
```

Comment:
same as above for this. + maybe we could add something to the readme or wiki on how to update the snapshots and why they might fail.

Reply:
Or even just link to the main repo snapshot documentation somewhere.

Reply:
Will add comments